src/commands/ext-configure.ts

```diff
@@ -69,6 +69,15 @@ export default new Command("ext:configure <instanceId>")
       const res = await extensionsApi.configureInstance(projectId, instanceId, params);
       spinner.stop();
       utils.logLabeledSuccess(logPrefix, `successfully configured ${clc.bold(instanceId)}.`);
+      utils.logLabeledBullet(
+        logPrefix,
+        marked(
+          `You can also view your newly configured instance in the Firebase console: ${utils.consoleUrl(
```

Comment:
Curious why the word "also" is used in this log?  (here and throughout)

Reply:
resolved with joehan@'s recent commits.  thx!

Reply:
Good point, removed.